### PR TITLE
add code generation space as a blog

### PR DESCRIPTION
**Pull request by loubnabnl**
Add this [space](https://huggingface.co/spaces/loubnabnl/code-generation-models) as a blog post.

cc @osanseviero

### CODE REVIEW

File: _blog.yml [hunk 1]
```diff
@@ -876,3 +876,13 @@
   tags:
   - guide
   - diffusion
+
+- external: https://huggingface.co/spaces/loubnabnl/code-generation-models
+  title: "Code generation with 🤗"
+  author: loubnabnl
+  thumbnail: /blog/assets/79_code_generation_space/code_generation.png
```
Comment (osanseviero) [inline _blog.yml:883]: I think the thumbnail could likely improve. @mishig25 @gary149 any ideas on this?

File: _blog.yml [hunk 2]
```diff
@@ -876,3 +876,13 @@
   tags:
   - guide
   - diffusion
+
+- external: https://huggingface.co/spaces/loubnabnl/code-generation-models
+  title: "Code generation with 🤗"
```
Comment (osanseviero) [inline _blog.yml:881]:
```suggestion
  title: "Code generation with Hugging Face"
```

**Review by osanseviero — APPROVED** [review]
Very nice! I think the thumbnail could be a bit more exciting, I'm cc'ing a couple of people to see if they have any ideas

File: _blog.yml (hunk 1 above)
Comment (julien-c) [reply _blog.yml:883]: what about just using the Space thumbnail for consistency? https://thumbnails.huggingface.co/social-thumbnails/spaces/loubnabnl/code-generation-models.png

Comment (lvwerra) [reply _blog.yml:883]: What do you think about something like this:

Screenshot 2022-06-08 at 13 51 49

Maybe we could highlight a bit more the difference between prompt and generation.

Comment (loubnabnl) [reply _blog.yml:883]: like this?
image

Comment (gary149) [reply _blog.yml:884]: I think using the [Space thumbnail](https://thumbnails.huggingface.co/social-thumbnails/spaces/loubnabnl/code-generation-models.png) is not a bad idea.

**Pull request merged**